pkg/sriov/policy.go

```diff
@@ -108,6 +112,10 @@ func (builder *PolicyBuilder) WithVFRange(firstVF, lastVF int) *PolicyBuilder {
 		return builder
 	}
 
+	if firstVF < 0 || lastVF < 0 {
+		builder.errorMsg = "firstPF and lastVF can not be less than 0"
```

Comment:
Based on condition this should be:
```suggestion
		builder.errorMsg = "firstPF or lastVF can not be less than 0"
```

Reply:
firstVF (not PF) according to the condition